web/pages/program/[slug].tsx

```diff
@@ -147,45 +192,17 @@ const SessionRoute = ({
                 )}
               </div>
 
-              {speakers && (
-                <ul className={programStyles.speakers}>
-                  {speakers.map(
-                    ({
-                      role,
-                      person: { _id, name, title, company, photo, slug },
-                    }) => (
-                      <li key={_id} className={programStyles.speakerItem}>
-                        <Link href={urlJoin('/speakers', slug.current)}>
-                          <a className={programStyles.speakerLink}>
-                            {photo && (
-                              /* eslint-disable-next-line @next/next/no-img-element */
-                              <img
-                                src={imageUrlFor(photo)
-                                  .size(64, 80)
-                                  .saturation(-100)
-                                  .url()}
-                                width={64}
-                                height={80}
-                                alt={name}
-                                className={programStyles.speakerImage}
-                              />
-                            )}
-                            <div>
-                              <div className={programStyles.role}>{role}</div>
-                              <strong className={programStyles.speakerName}>
-                                {name}
-                              </strong>
-                              <div>
-                                {[title, company].filter(Boolean).join(', ')}
-                              </div>
-                            </div>
-                          </a>
-                        </Link>
-                      </li>
-                    )
-                  )}
-                </ul>
-              )}
+              {speakers &&
+                (hasHighlightedSpeakers ? (
+                  <div className={programStyles.highlightedSpeakers}>
+                    <SessionSpeakers
+                      speaker1={speakers[0]?.person}
```

Comment:
```suggestion
                      speaker1={speakers[0].person}
```

(Super-nit, but since `hasHighlightedSpeakers` is true in this branch, `speakers[0]` can't be false. Perhaps then maybe we should remove the optional flag for `speaker1` in `<SessionSpeakers />` because it can't ever render with a falsy `speaker1`.)

Reply:
Technically speaking, `person` is apparently not a mandatory field, so it's possible to publish a session that results in us receiving something like `"speakers":[{"person":null,"role":"moderator"}]`.

Good point about `speakers[0]` not being falsy, though. I went through one or two iterations of the `hasHighlightedSpeakers` check, and this wasn't originally the case.